src/renderer/components/ContactList.js

```diff
@@ -51,9 +51,10 @@ class ContactList extends React.Component {
   }
 
   render () {
-    const { childProps, onContactClick } = this.props
+    const { childProps, onContactClick, sortFunc } = this.props
     let contacts = this.props.contacts
     if (!contacts) contacts = this._getContacts()
+    if (sortFunc) contacts = contacts.sort(sortFunc)
```

Comment:
Nice idea, but we don't really want to sort the contacts again. They are already sorted and there's one rule, never ever sort again things we get from the core, as they are already sorted. Sorting is slow, the fastest it can get is somewhere around `O n*log(n)`. We don't need to sort, we just need to pick out the contacts if they are in the group or not and keep the order of it. This would result in a time complexity of `O n` plus it would keep the actual alphabetical ordering.

Reply:
I replaced the sorting with a filter

Reply:
But we should probably use the right core methods in the future to get the members